modules/launch_utils.py

```diff
@@ -308,11 +308,11 @@ def requirements_met(requirements_file):
 
 
 def prepare_environment():
-    torch_index_url = os.environ.get('TORCH_INDEX_URL', "https://download.pytorch.org/whl/cu118")
-    torch_command = os.environ.get('TORCH_COMMAND', f"pip install torch==2.0.1 torchvision==0.15.2 --extra-index-url {torch_index_url}")
+    torch_index_url = os.environ.get('TORCH_INDEX_URL', "https://download.pytorch.org/whl/cu121")
+    torch_command = os.environ.get('TORCH_COMMAND', f"pip install torch==2.1.0 torchvision==0.16.0 --extra-index-url {torch_index_url}")

```

Comment:
does this need to upgraded as well? or does MPS still need this version of torch?
https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/master/webui-macos-env.sh#L14
```
export TORCH_COMMAND="pip install torch==2.0.1 torchvision==0.15.2"
```

Reply:
> does this need to upgraded as well? or does MPS still need this version of torch? https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/master/webui-macos-env.sh#L14
> 
> ```
> export TORCH_COMMAND="pip install torch==2.0.1 torchvision==0.15.2"
> ```

Yes it should be updated
I don't know if I change all the torch version parts

Maybe we need a singlr file for this kind of things